core/contracts/CentralizedStore.sol

```diff
@@ -18,6 +18,18 @@ contract CentralizedStore is StoreInterface, Withdrawable {
     uint private fixedOracleFeePerSecond; // Percentage of 10^18. E.g., 1e18 is 100% Oracle fee.
     uint private constant FP_SCALING_FACTOR = 10**18;
 
+    enum Roles {
+        Governance,
+        Writer,
+        Withdraw
+    }
+
+    constructor() public {
+        _createExclusiveRole(uint(Roles.Governance), uint(Roles.Governance), msg.sender);
+        createWithdrawRole(uint(Roles.Writer), uint(Roles.Governance), msg.sender);
+        createWithdrawRole(uint(Roles.Withdraw), uint(Roles.Withdraw), msg.sender);
```

Comment:
Should the second argument be `Roles.Governance`?

Reply:
ah yeah, good catch.